ci/pipeline

```diff
@@ -84,11 +84,11 @@ def checkSystemIntegrationTests(logFileName: String): Unit = {
  * Compresses sandboxes and logs.
  *
  * @param logFileName Name of log file.
- * @param sandboxFilePrefix The start of the file name for the sandboxes archive.
+ * @param sandboxFile The start of the file name for the sandboxes archive.
  */
 @main
-def zipLogs(logFileName: String = "ci.log", sandboxFilePrefix: String = "sandboxes"): Unit = {
-  Try(%("tar", "--exclude=provisioner", "-zcf", s"$sandboxFilePrefix.tar.gz", "sandboxes"))
+def zipLogs(logFileName: String = "ci.log", sandboxFile: String = "sandboxes"): Unit = {
```

Comment:
sandboxFileName? :) since it's not the file itself

Reply:
✅